pkg/util/i18n/translations/en-us.go

```diff
@@ -0,0 +1,16 @@
+package translations
+
+var EnUSFileName = "en-US.all.json"
+
+var EnUSData = []byte(`
```

Comment:
Is this really the best format available to us?  Java style resource bundle properties files seem less fragile.